Dockerfile.alpine

```diff
@@ -0,0 +1,60 @@
+FROM erlang:20.3-alpine AS build-env
+
+WORKDIR /tmp/vernemq-build
+
+ARG VERNEMQ_VERSION=1.6.0
+ARG TARGET=rel
+ARG VERNEMQ_REPO=https://github.com/vernemq/vernemq.git
+
+# Defaults
+ENV DOCKER_VERNEMQ_KUBERNETES_APP_LABEL vernemq
+ENV DOCKER_VERNEMQ_LOG__CONSOLE console
+
+RUN \
+    apk --no-cache --update --available upgrade \
+    && apk add --no-cache git autoconf build-base bsd-compat-headers cmake openssl-dev bash \
+    && git clone -b $VERNEMQ_VERSION --single-branch --depth 1 $VERNEMQ_REPO .
+
+ADD bin/build.sh build.sh
+
+RUN ./build.sh $TARGET
+
+FROM alpine
```

Comment:
Uhm, this is quite dangerous IMHO, given you're not giving any version boundary, and this Dockerfile is doomed to explode sometime in the future. I'd rather lock the Alpine version to the same version used by the builder image.